niworkflows/interfaces/images.py

```diff
@@ -632,7 +632,7 @@ class SignalExtractionInputSpec(BaseInterfaceInputSpec):
         desc='a 3-D label image, with 0 denoting '
         'background, or a list of 3-D probability '
```

Comment:
These can't be probability maps as they are (and were before this PR) cast to bool. These are masks, let's fix it now that we are at it. Let's add one more input to at least threshold at that value (better than > 0.0 as it currently works).

In the future we can remove the bool casting and use a weighted mean in line 690 instead (or you can tackle that now, which would be much better)

Reply:
At the moment I have added an input if probability threshold input which defaults at 0.5. But can change this to 0 if you want (which is what I think you are suggesting here).